### ISSUE TITLE
Ace Editor issues

### ISSUE THREAD

**Issue by mhgtlc** [pimcore/admin-ui-classic-bundle#110]
it seems ace editor does not work

i tried these options:
```
var jsonMode = ace.require("ace/mode/json").Mode;
editor.session.setMode(new jsonMode());
```
and
```
editor.session.setMode("ace/mode/json");
```

and it did not work

Also, are npm packages being built when imported from composer?

**Comment by dvesh3** [comment]
The editor works for me without the code highlight and search so seems like the build is incomplete. @robertSt7 could you please take a look into this?

**Comment by mhgtlc** [comment]
@robertSt7 , @dvesh3 when is this issue planned to be fixed? It has been moved from one milestone to another 5 times in a row :) from 1.0.1 to 1.0.6

**Comment by dvesh3** [comment]
Hi @mhgtlc this is low priority for us at the moment since there are more critical issues reported where features are broken with native type hint changes and also this is a cosmetic fix. we would be happy to receive a PR :)

**Comment by dvesh3** [comment]
Fixed by #207
